Gopkg.toml

```diff
@@ -98,7 +98,7 @@
 
 [[constraint]]
   name = "github.com/signalfx/golib"
-  revision = "93885a1b911579ca2ce6caf133916e0554e9ab37"
+  revision = "271e7b77c46c2cb201be901a302cb181a9d1645a"
```

Comment:
uhhh why do we have any changes at all to the Gopkg.toml or the vendor directory? It doesn't look like we're adding or changing any dependencies here.

Reply:
oh, the whole point of the PR was to update the sfx client for https://github.com/signalfx/golib/pull/104

Reply:
Ah, okay, mind adding that to the PR description?

Reply:
done